_includes/tutorials/kafka-producer-application/kafka/code/src/main/java/io/confluent/developer/KafkaProducerApplication.java

```diff
@@ -94,8 +92,10 @@ public static void main(String[] args) throws Exception {
             producerApp.printMetadata(metadata, filePath);
 
         } catch (IOException e) {
-            System.err.println(String.format("Error reading file %s due to %s", filePath, e));
+            System.err.printf("Error reading file %s due to %s", filePath, e);
```

Comment:
Minor nit - might consider adding `%n` (newline) to the end of the format string to match previous behavior.

Reply:
ack